src/Main.java

```diff
@@ -13,6 +14,17 @@ public static void main(String[] args) {
 		
 		System.out.println("【すべてのユーザーを表示する】");
 		users.forEach(u -> System.out.println("名前: %s, 生年月日: %s".formatted(u.getName(), u.getBirthDate())));
+	
+		System.out.println("【すべてのユーザーを表示する。ただし生年月日はyyyy/MM/dd(E)形式で出力する】");
+		users.forEach(u -> System.out.println("名前: %s, 生年月日: %s".formatted(u.getName(), japanFormat(u.getBirthDate()))));
+		
+		System.out.println("【名前が鈴木で始まる人のみを表示する】");
+		users.stream().filter(user ->user.getName().startsWith("鈴木"))
+						.forEach(u -> System.out.println(u.getName()));
+	}
+	
+	private static String japanFormat(LocalDate localDate) {
+		return localDate.format(DateTimeFormatter.ofPattern("yyyy年MM月dd日(E)").localizedBy(Locale.JAPAN));
```

Comment:
DateTimeFormatterを使っていていいですね 👍 
今回だとlocalizedByはいらないかなと思います。
```suggestion
		return localDate.format(DateTimeFormatter.ofPattern("yyyy年MM月dd日(E)"));
```